src/video/uikit/SDL_uikitwindow.m

```diff
@@ -370,6 +375,13 @@ - (void)layoutSubviews
                     info->info.uikit.colorbuffer = glview.drawableRenderbuffer;
                     info->info.uikit.resolveFramebuffer = glview.msaaResolveFramebuffer;
                 } else {
+#elif SDL_VIDEO_METALANGLE
```

Comment:
If the metal-angle view code is able to be merged into ` SDL_uikitopenglview` using a couple defines in key places, this codepath shouldn't be needed anymore.